fog/test-client/src/config.rs

```diff
@@ -71,6 +76,22 @@ pub struct TestClientConfig {
     /// Amount to transfer per transaction
     #[structopt(long, default_value = "20")]
     pub transfer_amount: u64,
+
+    /// Consensus enclave CSS file (overriding the build-time CSS)
+    #[structopt(long)]
```

Comment:
Since you're adding new options to the CLI, might it be worth testing using assert_cmd? Or, in general, what is the testing strategy for this PR? I understand that this updates the test client, so it might just rely on preexisting tests, but just want to double check.

Reply:
it's kind of hard to test this without standing up the whole system, and the test-client itself is test code. so i think the idea is basically, we run it and we see if it works, so, running it through CD (which doesn't test the continuous mode but tests that we didn't break the previous mode), and then running it IRL against testnet manually.

we could try to add the continuous mode to the dev networks and so get more coverage that way, and that would be good because we would be able to get these metrics in grafana for the test networks. but that is work in the internal repo.

Reply:
we could plausibly get coverage in CI for this by doing like, start a network in CI using fog local network and then running this, and that would be a way to test that the fog local network script is still working.

i guess i tend to think that, when it's an artifact that we don't ship, that is just test code, it doesn't need test coverage, running it is the test, and if we run it in CD or in testnet and it's broken, then we'll iterate on it and fix it at that time

Reply:
many types of changes are supposed to run through CD in the dev networks before merge though, so if we made a change that somehow broke the test client, that would surface it before merge

Reply:
Thanks for the explanation. And yes, I agree that since the issue with the test client would surface before merge, then it should be ok to not have an explicit test for it.